OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -4,6 +4,7 @@
 import android.annotation.SuppressLint;
 import android.app.Activity;
 import android.content.Intent;
+import android.os.Build;
```

Comment:
Resolve warnings if possible

Reply:
File will be converted in Kotlin for AppNav branch. Overhead.